hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -1410,6 +1410,13 @@
       en_run_modes: ["sw_test_mode_test_rom"]
       run_opts: ["+en_scb=0", "+bypass_alert_ready_to_end_check=1"]
     }
+    {
+      name: chip_sw_rv_dm_access_after_wakeup
```

Comment:
Please map this test to the entry in the testplan.

Reply:
done :+1: